project_national/fact_sheets_category_4.yml

```diff
@@ -0,0 +1,318 @@
+        apply_logic:
+          - and:
+            - HVAC Has Ducts|No
+            - HVAC Cooling Type|None
+      - &eff_central_ac_southeast_and_southwest
+        option: HVAC Cooling Efficiency|AC, SEER 15
+        apply_logic:
+          - and:
+            - HVAC Has Ducts|Yes
+            - HVAC Cooling Type|None
+            - or:
+              - State|CA
+              - State|NV
+              - State|AZ
+              - State|NM
+              - State|OK
+              - State|TX
+              - State|AR
+              - State|LA
+              - State|KY
+              - State|TN
+              - State|MS
+              - State|AL
+              - State|GA
+              - State|FL
+              - State|SC
+              - State|NC
+              - State|VA
+              - State|DC
+              - State|MD
+              - State|DE
+      - &eff_central_ac_us_others
+        option: HVAC Cooling Efficiency|AC, SEER 14
+        apply_logic:
+          - and:
+            - HVAC Has Ducts|Yes
+            - HVAC Cooling Type|None
+            - or:
+              - State|WA
+              - State|OR
+              - State|MT
+              - State|ID
+              - State|WY
+              - State|UT
+              - State|CO
+              - State|ND
+              - State|SD
+              - State|NE
+              - State|KS
+              - State|MN
+              - State|IA
+              - State|MO
+              - State|WI
+              - State|IL
+              - State|MI
+              - State|IN
+              - State|OH
+              - State|WV
+              - State|PA
+              - State|NJ
+              - State|NY
+              - State|CT
+              - State|RI
+              - State|MA
+              - State|VT
+              - State|NH
+              - State|ME
+
+  - upgrade_name: ENERGYSTAR Room AC
+    options:
+      - &energystar_room_ac
+        option: HVAC Cooling Efficiency|Room AC, EER 12.0
+        apply_logic:
+          - or:
+            - HVAC Cooling Efficiency|Room AC, EER 8.5
+            - HVAC Cooling Efficiency|Room AC, EER 9.8
+            - HVAC Cooling Efficiency|Room AC, EER 10.7
+
+  - upgrade_name: ENERGYSTAR Central AC
+    options:
+      - &energystar_central_ac
+        option: HVAC Cooling Efficiency|AC, SEER 15.2
+        apply_logic:
+          - or:
+            - HVAC Cooling Efficiency|AC, SEER 8
+            - HVAC Cooling Efficiency|AC, SEER 10
+            - HVAC Cooling Efficiency|AC, SEER 13
+            - HVAC Cooling Efficiency|AC, SEER 15
+
+  - upgrade_name: HPWH
+    options:
+      - &water_heater_fuel_type
+        option: Water Heater Fuel|Electricity
+        apply_logic:
+          - not: Water Heater Fuel|Electricity
+      - &water_heater_efficiency_electric_heat_pump_50_gal_3_pt_45_uef
+        option: Water Heater Efficiency|Electric Heat Pump, 50 gal, 3.45 UEF
+        apply_logic:
+          - and:
```

Comment:
I don't believe the `- and:` is needed here as `and:` is default

Reply:
done